src/main/resources/feedbackQuestionConstSumEditFormOptionFragment.html

```diff
@@ -1,5 +1,5 @@
-<div id="constSumOptionRow-${i}-${questionNumber}">
-    <div class="input-group col-sm-12">
+<div class="margin-bottom-7px" id="constSumOptionRow-${i}-${questionNumber}">
```

Comment:
Hmm. We already seem to have a few such classes already (margin-bottom-15px, etc.). Moreover, I think making it question specific makes it less reusable in the future?